crates/solidity/testing/snapshots/cst_output/DeleteStatement/delete_identifier/generated/0.4.11.yml

```diff
@@ -7,8 +7,9 @@ Errors: []
 
 Tree:
   - DeleteStatement (Rule): # 0..11 "delete foo;"
-      - DeleteKeyword (Token): "delete" # 0..6
-      - Expression (Rule): # 7..10 "foo"
-          - PrimaryExpression (Rule): # 7..10 "foo"
-              - Identifier (Token): "foo" # 7..10
-      - Semicolon (Token): ";" # 10..11
+      - _TERMINATEDBY (Rule): # 0..11 "delete foo;"
```

Comment:
I wonder if we should produce a separate node for an unnamed `_TERMINATEDBY` node. I imagine for cases like `DeleteStatement`, we would want to hide it, maintaining the original tree shape here.

Reply:
_DELIMITEDBY and _TERMINATEDBY can be inlined into the parent because they have a fixed arity. But _SEPARATEDBY is like _REPEATED i.e it has variable arity.

Reply:
(done)

Reply:
Ah yes! Thanks.

Reply:
Is there any remaining use case for keeping `_TERMINATEDBY` and `_DELIMITEDBY` token kinds?

Reply:
Actually, that is probably a more involving change. Let's move the discussion to #353